.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: test
+on: push
+
+jobs:
+    pypi-publish:
+        if: startsWith(github.ref, 'refs/tags/')
```

Comment:
this makes sure that this runs only when a tag is pushed, right?

Reply:
Yes, I couldn't find a more robust way to do it, but it's the suggested way in the action documentation and in the [python guide](https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#the-whole-ci-cd-workflow).

Reply:
great, adjusted it a bit based on the action's recommendations
![Uploading image.png…]()